app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -77,6 +77,15 @@
   meta_tags["govuk:taxon-ids"] = taxon_ids.join(',') unless taxon_ids.empty?
   meta_tags["govuk:taxon-slug"] = taxon_slugs_without_theme.first unless taxon_slugs_without_theme.empty?
   meta_tags["govuk:taxon-slugs"] = taxon_slugs_without_theme.join(',') unless taxon_slugs_without_theme.empty?
+
+  def has_content_history(content_item)
```

Comment:
Presumably we have to embed methods and conditional logic here because this is a `raw.html.erb` template? :cry: